FirebasePerformance/Sources/FPRClient.m

```diff
@@ -48,17 +52,21 @@ @interface FPRClient ()
 @implementation FPRClient
 
 + (void)load {
-  __weak NSNotificationCenter *notificationCenter = [NSNotificationCenter defaultCenter];
-  __block id listener;
-
-  void (^observerBlock)(NSNotification *) = ^(NSNotification *aNotification) {
-    NSDictionary *appInfoDict = aNotification.userInfo;
-    NSNumber *isDefaultApp = appInfoDict[kFIRAppIsDefaultAppKey];
-    if (![isDefaultApp boolValue]) {
-      return;
+  [FIRApp registerInternalLibrary:[FPRClient class]
+                         withName:@"fire-perf"
+                      withVersion:[NSString stringWithUTF8String:kFPRSDKVersion]];
```

Comment:
Can kFPRSDKVersion ever be different than Firebase Version? I think this line should be deleted an kFPRSDKVersion be removed.

Reply:
Reference to the constant value of SDK version https://github.com/firebase/firebase-ios-sdk/blob/master/FirebasePerformance/Sources/Common/FPRConstants.m#L22